pkg/controllers/openshift-apiserver.go

```diff
@@ -230,14 +225,17 @@ apiServerArguments:
   - Scope
   - SystemMasters
   - RBAC
-  - Node
+  - Node`)
+
+	if cfg.AuditLogDir != "" {
+		data = append(data, `
 auditConfig:
-  auditFilePath: "` + cfg.LogDir + `/openshift-apiserver/audit.log"
+  auditFilePath: `+cfg.AuditLogDir+`openshift-apiserver-audit.log
```

Comment:
```suggestion
  auditFilePath: "`+cfg.AuditLogDir+`openshift-apiserver-audit.log"
```

should we keep the commas?